### PR TITLE
Create answer to speech documentation

### PR DESCRIPTION
**Pull request by agnieszka-m**

### CODE REVIEW

**Comment by agnieszka-m** [comment]
Hey @ZanSara , can you please have a look if that covers it or do we need something more? Thanks!

File: docs/latest/pipeline_nodes/document_to_speech.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+answer_dir = './generated_audio_answers'
+
+audio_document = DocumentToSpeech(model_name_or_path=model_name, generated_audio_dir=answer_dir)
+```
+To use `DocumentToSpeech` in a pipeline, run:
+```python
+from haystack.nodes import DocumentToSpeech
+
+retriever = BM25Retriever(document_store=document_store)
+reader = FARMReader(model_name_or_path="deepset/roberta-base-squad2", use_gpu=True)
```
Comment (ZanSara) [inline docs/latest/pipeline_nodes/document_to_speech.mdx]: The reader is not needed in document retrieval. A doc retrieval pipeline in its simplest form has only a retriever node.

**Review by ZanSara — CHANGES_REQUESTED** [review]
One thing to fix in the DocumentToSpeech part, otherwise good!

File: docs/latest/pipeline_nodes/document_to_speech.mdx (hunk 1 above)
Comment (agnieszka-m) [reply docs/latest/pipeline_nodes/document_to_speech.mdx]: Ah, right, fixed :)

**Comment by ZanSara** [comment]
Note: I realized updating the tutorials that `DocumentToSpeech` can fit well into 2 positions in a pipeline:

1. At the end of a Document Search pipeline (single-retriever pipeline)
2. At the end of an indexing pipeline (right before the Document Store)

I found no way to add this information in the template though. An example indexing pipeline with a DocumentToSpeech looks like:
```python
file_paths = [p for p in Path(documents_path).glob("**/*")]

indexing_pipeline = Pipeline()

classifier = FileTypeClassifier()
indexing_pipeline.add_node(classifier, name="classifier", inputs=["File"])

text_converter = TextConverter(remove_numeric_tables=True)
indexing_pipeline.add_node(text_converter, name="text_converter", inputs=["classifier.output_1"])

preprocessor = PreProcessor(
        clean_whitespace=True,
        clean_empty_lines=True,
        split_length=100,
        split_overlap=50,
        split_respect_sentence_boundary=True,
)
indexing_pipeline.add_node(preprocessor, name="preprocessor", inputs=["text_converter"])

doc2speech = DocumentToSpeech(model_name_or_path="espnet/kan-bayashi_ljspeech_vits", generated_audio_dir=Path("./audio_documents"))
indexing_pipeline.add_node(doc2speech, name="doc2speech", inputs=["preprocessor"])

document_store = ElasticsearchDocumentStore(host="localhost", username="", password="", index="document")
indexing_pipeline.add_node(document_store, name="document_store", inputs=["doc2speech"])

indexing_pipeline.run(file_paths=file_paths, meta=files_metadata)
```

**Review by ZanSara — APPROVED** [review]

**Pull request merged**